blog/_posts/2016-12-xx-moredots.md

```diff
@@ -0,0 +1,600 @@
+---
+layout: post
+title:  More Dots: Syntactic Loop Fusion in Julia
+author: <a href="http://math.mit.edu/~stevenj">Steven G. Johnson</a>
+---
+
+After a [lengthy design process](https://github.com/JuliaLang/julia/issues/8450) and [preliminary foundations in Julia 0.5](http://julialang.org/blog/2016/10/julia-0.5-highlights#vectorized-function-calls), Julia 0.6 includes new facilities for writing code in the "vectorized"
+style (familiar from Matlab, Numpy, R, etcetera) while avoiding the
+overhead that this style of programming usually imposes: multiple
+vectorized operations can now be "fused" into a single loop, without
+allocating any extraneous temporary arrays.
```

Comment:
"any arrays" seems more accurate, and is grammatical.   Typically, more than one temporary array would be allocated without fusion.